src/www/app/fake_tunnel.ts

```diff
@@ -33,14 +33,14 @@ export class FakeOutlineTunnel implements Tunnel {
     return name?.toLowerCase().includes('unreachable');
   }
 
-  async start(config: ShadowsocksConfig): Promise<void> {
+  async start(config: ShadowsocksSessionConfig): Promise<void> {
     if (this.running) {
       return;
     }
 
-    if (this.playUnreachable(config.name)) {
+    if (this.playUnreachable(config.tag)) {
```

Comment:
Let's change this to be based on the hostname instead. I think this is the only dependency on the name

Reply:
Er, I changed this in the load as well but I'm not sure that's intended: https://github.com/Jigsaw-Code/outline-client/pull/1401/files#diff-6271a745a4aa22be61223a320663a22548ada7dd5f593c6cbd6efb74dcf45c41R201

Reply:
Oh you know what, I know what to do